src/engine/QueryPlanner.cpp

```diff
@@ -498,7 +498,8 @@ QueryPlanner::TripleGraph QueryPlanner::createTripleGraph(
   vector<const SparqlTriple*> entityTriples;
   // Add one or more nodes for each triple.
   for (auto& t : pattern->_triples) {
-    if (t.p_.iri_ == CONTAINS_WORD_PREDICATE) {
+    if (std::holds_alternative<PropertyPath>(t.p_) &&
+        std::get<PropertyPath>(t.p_).iri_ == CONTAINS_WORD_PREDICATE) {
```

Comment:
With a simple `t._p.isPredicateIri(Contains_word_predicate)` this complicated functions become simpler.